test/cmd/create.sh

```diff
@@ -83,13 +83,13 @@ run_create_job_tests() {
     # Test kubectl create job
     kubectl create job test-job --image=k8s.gcr.io/nginx:test-cmd
     # Post-Condition: job nginx is created
-    kube::test::get_object_assert 'job test-job' "{{$image_field0}}" 'k8s.gcr.io/nginx:test-cmd'
+    kube::test::get_object_assert 'job test-job' "{{${image_field0:?}}}" 'k8s.gcr.io/nginx:test-cmd'
     # Clean up
     kubectl delete job test-job "${kube_flags[@]}"
 
     # Test kubectl create job with command
     kubectl create job test-job-pi "--image=$IMAGE_PERL" -- perl -Mbignum=bpi -wle 'print bpi(20)'
-    kube::test::get_object_assert 'job test-job-pi' "{{$image_field0}}" $IMAGE_PERL
+    kube::test::get_object_assert 'job test-job-pi' "{{$image_field0}}" "$IMAGE_PERL"
```

Comment:
what is IMAGE_PERL?

Reply:
https://github.com/kubernetes/kubernetes/blob/master/test/cmd/legacy-script.sh#L72
It's defined in the `legacy-script.sh` as `IMAGE_PERL="k8s.gcr.io/perl"`